libs/base/utilities/src/misc/events-repeater.ts

```diff
@@ -0,0 +1,97 @@
+export const EVENTS_DATA = Symbol.for('EVENTS_DATA');
+export const replayableAttribute = 'replayable';
+
+export type ElementWithEventsData = HTMLElement & {
+  [EVENTS_DATA]?: EventsData;
+};
+
+interface EventsData {
+  listener: (event: Event) => void;
+  events: Event[];
+}
+
+const treewalk = (
```

Comment:
Wondering if this should be the (partially) same code that is used in the index.html

Reply:
when we move it to separate module I planned to reuse this code